datafusion/physical-expr-common/src/binary_view_map.rs

```diff
@@ -149,7 +149,7 @@ where
             output_type,
             map: hashbrown::raw::RawTable::with_capacity(INITIAL_MAP_CAPACITY),
             map_size: 0,
-            builder: GenericByteViewBuilder::new(),
+            builder: GenericByteViewBuilder::new().with_block_size(2 * 1024 * 1024),
```

Comment:
After https://github.com/apache/arrow-rs/pull/6136 is merged, will this be needed anymore?